package wrm.progetto_wrm.UTILITIES.Exceptions;

public class TaskAlreadyExistsException extends RuntimeException{
    public TaskAlreadyExistsException () {}
}
